lib/sandthorn_sequel_projection.rb

```diff
@@ -22,7 +21,7 @@ class << self
     require 'delegate'
     extend Forwardable
 
-    def_delegators :configuration, :batch_size, :event_stores
+    def_delegators :configuration, :batch_size, :event_store
```

Comment:
It's not per projection, a projection has only one. It is `SandthornSequelProjection` that has the ability to handle projections pointing at different event stores.